CrudeOperation/src/main/java/com/example/crudeoperation/controller/DeptController.java

```diff
@@ -0,0 +1,46 @@
+package com.example.crudeoperation.controller;
+
+
+import com.example.crudeoperation.entity.DeptEntity;
+import com.example.crudeoperation.service.DeptService;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@CrossOrigin("*")
+
+public class DeptController {
+   private final DeptService serviceObj;
+
+    public DeptController(DeptService serviceObj) {
+        this.serviceObj = serviceObj;
+    }
+
+    // call all crud methods one by one
+    @PostMapping("saveData")
+    public DeptEntity saveDeptInfo(@RequestBody DeptEntity Department) {
```

Comment:
don't use entity class in request body create DTO class for that

Reply:
ok